src/scripts/OSFramework/Helper/Device.ts

```diff
@@ -127,9 +127,16 @@ namespace OSFramework.Helper {
 		 * @memberof DeviceInfo
 		 */
 		private static _getUserAgent(userAgent = ''): string {
-			return userAgent.replace(' ', '') === ''
-				? window.navigator.userAgent.toLowerCase()
-				: userAgent.toLowerCase();
+			const cleanedUserAgent = userAgent.replace(' ', '');
+
+			if (cleanedUserAgent === '') {
+				if (sessionStorage.userAgent) {
+					return sessionStorage.userAgent.toLowerCase();
```

Comment:
The name of the variable in the session storage, should be more... explicit. Something like `sessionStorage.previewDevicesUserAgent`...

Reply:
Done